background.js

```diff
@@ -5,12 +5,18 @@ window.updatePopup = null;
 let webPageConnection = null;
 let socket = null;
 let roomId = null;
+let skipIntro = true;
 
 loadStyles();
 
-chrome.tabs.onActivated.addListener(({tabId}) => getExtensionColor().then(color => setIconColor(tabId, color )));
+chrome.tabs.onActivated.addListener(({tabId}) => {
+  getExtensionColor().then(color => setIconColor(tabId, color ))
+  getIntroFeatureState().then(state => skipIntro = state);
+});
 
 chrome.tabs.onUpdated.addListener(function (tabId, changeInfo, tab) {
+  getIntroFeatureState().then(state => skipIntro = state);
+  console.log({ skipIntro });
```

Comment:
It might be better to use the debug function, so those logs won't go to production

Reply:
Actually, I forgot to remove this.